sweagent/environment/swe_env.py

```diff
@@ -216,8 +220,13 @@ def communicate(
         """
         self.logger.log(logging.TRACE, "Input:\n%s", input)  # type: ignore
         rex_check = "silent" if check else "ignore"
+
+        bash_lines = [f'export ENV_DIR="{env_dir}"', f'export REPO_BASE_DIR="{repo_base_dir}"', input]
```

Comment:
Oh I hadn't noticed that this was in `communicate`. This should definitely be moved to some startup commands, rather than being appended to every single communicate call.

Reply:
Sure, I’ll do that if the new PR is worthwhile.